src/styles/blocks/card.scss

```
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.product {
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+
+  &__photo {
+    width: 100%;
+    height: 134px;
+    margin-bottom: 36px;
+    padding: 0 3px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-accent;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin-bottom: 16px;
+
+  }
+
+  &__rank {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    height: 16px;
+
+    &-reviews {
+      color: $main-accent;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    height: 18px;
+
+    &-value {
+      color: $main-accent;
+      text-align: right;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+}
+
+.product__btn {
+  display: inline-block;
+  text-align: center;
+  width: 100%;
+  padding: 12px 70px 12px 69px;
+  flex-shrink: 0;
+  border-radius: 5px;
+  color: #FFF;
+  background: $blue-accent;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px; /* 114.286% */
```

Remove comments everywhere
```suggestion
  line-height: 16px;
```